### PR TITLE
Add links to social media for speakers

### LINKED ISSUE #27: Speaker social icon-links aren't initialized
Twitter, GitHub, and LinkedIn icon links are just redirecting to the same page:

screen shot 2015-05-15 at 3 44 49 pm

### PR DESCRIPTION
**Pull request by nathany**
Fixes #27 

If I couldn't find a person or wasn't sure, I left the the link blank so that icon doesn't show. This pull request also adds social icons to the /speakers page and hides the icons when no link is available.

### TODO:

* [ ] Add missing speaker links
* [ ] Verify existing speakers
* [ ] Exclude any links speakers don't wish to share (remove line entirely)

### Speakers:

* [x] Abhishek Kona @sheki
* [x] Andrew Gerrand @adg
* [x] Audrey Lim @audreylim
* [x] Baishampayan Ghose @ghoseb
* [x] Barak Michener @barakmich
* [x] Ben Johnson @benbjohnson
* [x] Björn Rabenstein @beorn7
* [x] Blake Caldwell @wblakecaldwell
* [x] Derek Parker @derekparker
* [x] Dmitry Vyukov @dvyukov
* [x] Hana Kim @hyangah
* [x] Katherine Cox-Buday @kat-co
* [x] Kelsey Hightower @kelseyhightower
* [x] Kevin Cantwell @kevin-cantwell
* [x] Kyle Erf @3rf
* [x] Peter Bourgon @peterbourgon
* [x] Richard Fliam @rfliam
* [x] Rick Hudson @RLH
* [x] Robert Griesemer @griesemer
* [x] Russ Cox @rsc
* [ ] Sam Helman @shelman
* [x] Sarah Adams @adams-sarah
* [x] Tomás Senart @tsenart

### CODE REVIEW

**Comment by nathany** [comment]
@bketelsen @erikstmartin Do you have, or could you obtain the information for the other speakers?

**Comment by bketelsen** [comment]
tweeted a request for speakers to add their own.

BK

File: content/speakers/peter-bourgon.md [hunk 1]
```diff
@@ -6,5 +6,8 @@ linktitle = "Peter Bourgon"
 title = "Peter Bourgon"
 speakerimage = "/images/speakers/peter-bourgon.jpg"
 talk="/talks/go-kit"
+twitter="https://twitter.com/peterbourgon"
+github="https://github.com/peterbourgon"
+linkedin="https://de.linkedin.com/pub/peter-bourgon/15/26a/9a0"
```
Comment (peterbourgon) [inline content/speakers/peter-bourgon.md]: Please don't list my LinkedIn. Thank you!

**Comment by nathany** [comment]
Thanks Brian. 

Note: I expect to be offline most of tomorrow, so if someone else can finalize this pull request, that'd be great.

```
git remote add libgo https://github.com/libgo/gc15.git
```

File: content/speakers/peter-bourgon.md (hunk 1 above)
Comment (nathany) [inline content/speakers/peter-bourgon.md]: Okay. Will update it now.

**Comment by barakmich** [comment]
All correct for me.

**Comment by derekparker** [comment]
Looks good as far as mine go.

**Comment by bketelsen** [comment]
Also need to allow speakers to filter the media links they DON'T want to share.

**Comment by beorn7** [comment]
Looks good from my side. (Indeed no twitter...)

**Comment by 3rf** [comment]
Looks good to me

**Comment by sheki** [comment]
Lgtm

**Comment by adg** [comment]
LGTM

**Comment by kat-co** [comment]
LGTM

**Comment by audreylim** [comment]
LGTM

**Comment by tsenart** [comment]
:shipit:

**Comment by wblakecaldwell** [comment]
LGTM - I created a pull request to libgo/gc15:social with my LinkedIn link - Thanks!

**Comment by RLH** [comment]
LGTM

**Comment by nathany** [comment]
Thanks @wblakecaldwell. Merged your change in.

Listing links here is fine too.

**Comment by adams-sarah** [comment]
Mine are good! :ship:

**Comment by hyangah** [comment]
LGTM

**Comment by dvyukov** [comment]
lgtm

**Comment by rsc** [comment]
LGTM

**Comment by benbjohnson** [comment]
:+1:

**Comment by ghoseb** [comment]
LGTM :+1:

**Comment by griesemer** [comment]
LGTM (and its @robertgriesemer as in the change - NOT @griesemer, for Twitter).

**Comment by kelseyhightower** [comment]
LGTM

**Comment by rfliam** [comment]
LGTM

My linked in is https://www.linkedin.com/pub/richard-fliam/32/691/345 and my twitter is @zbobet2012

**Comment by davecheney** [comment]
@nathany thanks for putting this together. Let's meet up early today and get this merged.

**Comment by nathany** [comment]
@davecheney I haven't found any contact information for Sam Helman to confirm his info. :frowning: 

Other than that, is anything else needed? Is the actual code change good? Want me to squash this down to 2-3 commits?

**Comment by bketelsen** [comment]
We can always add more information later - better to get this merged now

**Comment by davecheney** [comment]
@nathany @bketelsen nah, i don't think we care about repo history here; we're all about action at Gophercon. 

I checked this out locally, looks fine. Gonna merge this now. 

We can pick up the stragglers in a later change.

**Comment by nathany** [comment]
Thanks @davecheney

**Pull request merged**